### PR TITLE
Remove mentions of attributes

### PR DESCRIPTION
**Pull request by damacus**
### Description

[Describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves]

### Check List
- [ ] All tests pass. See https://github.com/chef-brigade/sysctl/blob/master/TESTING.md
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

File: libraries/helpers_param.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 module SysctlCookbook
   module SysctlHelpers
     module Param
+      # turn into proerty
```
Comment (tas50) [inline libraries/helpers_param.rb]: typo

File: metadata.rb [hunk 2]
```diff
@@ -7,7 +7,7 @@
 description 'Configures sysctl parameters'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version '0.10.2'
-chef_version '>= 12.5' if respond_to?(:chef_version)
+chef_version '>= 13.0' if respond_to?(:chef_version)
```
Comment (tas50) [inline metadata.rb]: ?

File: test/cookbooks/test/metadata.rb [hunk 3]
```diff
@@ -1,4 +1,4 @@
-name 'sysctl_test'
+name 'test'
 maintainer 'Viverae, Inc.'
```
Comment (tas50) [inline test/cookbooks/test/metadata.rb]: we should just nuke the maintainer stuff since this is just a test recipe

**Comment by mkristan** [comment]
May I ask why are attributes removed from this cookbook?  I'm trying to understand the design rationale.

**Comment by damacus** [comment]
I’ll write up a blog post on this, as I keep getting asked!

the TLDR is, they’re compose-able resources just like normal Chef resources, and should be easier to reason with.

**Pull request merged**